core/migrations/0004_copyright_lccndatecopyright.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0003_auto_load_fixtures'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Copyright',
+            fields=[
+                ('uri', models.CharField(max_length=100, serialize=False, primary_key=True)),
```

Comment:
I'd recommend using an auto-increment integer as the primary key here as well. I do that for all non-joining tables as best practice. Using the smaller, unsigned int data types if possible. Not sure if django migrations handle that. We were working on a Rails project and it was having none of that deeper optimization.

But I would add a unique index to this uri field for data integrity :+1:

Smaller primary keys are better, second bullet point: https://dev.mysql.com/doc/refman/5.7/en/optimizing-innodb-storage-layout.html